### PR TITLE
File uploading endpoint added

### PR DESCRIPTION
**Pull request by SaptakS**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 

#3738

### CODE REVIEW

File: docs/api/api_blueprint.apib [hunk 1]
```diff
@@ -6530,3 +6530,25 @@ A temporary cropped image is first uploaded using this endpoint. After saving th
...
+
+
+### Upload a File [POST]
+
++ Parameters
+    + force_local (optional, string, `true`) - Force storage of temporary image in local
+
++ Request(multipart/form-data; boundary=---BOUNDARY)
+
+        -----BOUNDARY
```
Comment (niranjan94) [inline docs/api/api_blueprint.apib]: Add sample data too. Blueprint should have a full sample request

Comment (SaptakS) [inline docs/api/api_blueprint.apib]: @niranjan94 as far as I saw in examples of API Blueprint of files, if the data is in the form of base64encoded image or any other hash, then data is also included. Else, it is shown like this. One of the links I referred is [this](https://github.com/apiaryio/dredd/issues/129#issue-54313115). See the CSV file upload in the example.

Comment (niranjan94) [inline docs/api/api_blueprint.apib]: Hmmm ... okay

Comment (SaptakS) [inline docs/api/api_blueprint.apib]: Anything else?

File: app/api/uploads.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        )
+    else:
+        image_url = upload(
+            image_file,
+            UPLOAD_PATHS['temp']['image'].format(uuid=uuid.uuid4())
+        )
+    return jsonify({"url": image_url})
+
+
+@upload_routes.route('/file', methods=['POST'])
```
Comment (niranjan94) [inline app/api/uploads.py]: lets call it `files` by default

Comment (SaptakS) [inline app/api/uploads.py]: Ok. I saw the issue description and did accordingly. 😅

Comment (SaptakS) [inline app/api/uploads.py]: Resolved.

**Pull request merged**